src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +61,16 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    /**
+     * Comparator for sorting people by alphabetical order.
+
+    public static Comparator<Name> nameComparator = new Comparator<Name>() {
+        @Override
+        public int compare(Name p1, Name p2) {
+            String personName1 = p1.getWordsInName().get(0);
+            String personName2 = p2.getWordsInName().get(0);
+            return personName1.compareTo(personName2);
```

Comment:
What if their first names are the same but their last names are not? For example, Sarah Lim and Sarah Goh.